consult.el

```diff
@@ -76,10 +76,10 @@
   :type 'function)
 
 (defcustom consult-mode-histories
```

Comment:
I think this makes more sense as a `defvar` than as a `defcustom`.

Reply:
No, everything which is meant to be customizable should be defcustom. defvar is used for internal variables and more expert variables which are not meant to be customized, e.g. consult--gc-percentage.